features/step_definitions/fincap_pages_steps.rb

```diff
@@ -0,0 +1,67 @@
+Given(/^I have an insight page layout$/) do
+  english_site = Comfy::Cms::Site.find_or_create_by(
```

Comment:
We already have factories for creating the English site. I anticipate that this will change in the future. 
It's fine to have it here, as it is, for now though.